### PR TITLE
feat(nwp): add ingress from cluster-monitoring to applicationnamespace

### PR DESCRIPTION
**Pull request by zdtsw**
## Description

ref: https://issues.redhat.com/browse/RHOAIENG-3116

live build:  quay.io/wenzhou/opendatahub-operator-catalog:v2.8.3116

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by zdtsw** [comment]
@astefanutti need double check if we need the opening of `openshift-user-workload-monitoring` as well?

File: controllers/dscinitialization/utils.go [hunk 1]
```diff
@@ -286,6 +290,28 @@ func (r *DSCInitializationReconciler) reconcileDefaultNetworkPolicy(ctx context.
 							},
 						},
 					},
+					{
+						From: []netv1.NetworkPolicyPeer{
+							{ // need this for UWM work: cluster-monitoring->ODH namespaces
```
Comment (astefanutti) [inline controllers/dscinitialization/utils.go:295]: UWM -> CM (cluster monitoring)

**Review by astefanutti — APPROVED** [review]

**Comment by zdtsw** [comment]
@bartoszmajsak wanna also hear what you think

my plan is to fill the missing pieces which exists in downstream into ODH first.
then we can remove the ones in downstream
=> the first goal to use the same rules/code for both ODH and downstream

the next goal is to have agreement in the project: which default nwp that operator provides and what component can add/customize upon that or not

**Review by VaishnaviHire — APPROVED** [review]
/lgtm

**Pull request merged**